docs/metrics.rst

```diff
@@ -4,7 +4,7 @@ Metrics
 Metrics Records
 ---------------
 
-At the end of a race, Rally stores all metrics records in its metrics store, which is a dedicated Elasticsearch cluster. Rally stores the metrics in the indices ``rally-metrics-*``. It will create a new index for each month.
+At the end of a race, Rally stores all metrics records in its metrics store, which is a dedicated Elasticsearch cluster, ie. not the cluster where Rally ran its benchmarks. Rally stores the metrics in the indices ``rally-metrics-*``. It will create a new index for each month.
```

Comment:
Perhaps worth adding a link here to the configuration section on reporting/metrics (https://esrally.readthedocs.io/en/stable/configuration.html#reporting) ?

Reply:
Good idea, will do

Reply:
I tried to use Sphinx's internal links, but failed to link to the subsection, so I linked directly to the online docs.

Please take another look!